developer-guides/graphql-api/authentication/README.md

```diff
@@ -67,12 +67,7 @@ Imagine a situation where Rocket.Chat, the main Meteor application exposes only
 
 Grant also exposes an endpoint that shows which providers are enabled and used so you can make only those visible in your application.
 
-We also created a few packages to have something to start with: <br/>
-[`rocketchat:grant-facebook`](https://github.com/RocketChat/Rocket.Chat/tree/develop/packages/rocketchat-grant-facebook) <br/>
```

Comment:
I don't think that we should remove these links, they are only with the invalid link. These packages were moved to the `app` folder, the correct path is, for example: [`grant-facebook`](https://github.com/RocketChat/Rocket.Chat/tree/develop/app/grant-facebook). Maybe we should just change the links to the correct ones. @lolimay what do you think?

Reply:
Yes, you are right. It's my fault 😅

Reply:
Sorry for the delay! I've fixed the invalid links, please review :)